doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Materialize Kubernetes Operator"
+description: ""
+
+---
```

Comment:
We could definitely make this an overview page.
And move the content to an Install on AWS page as well.   But, at least, wanted to give an idea about the content.